### ISSUE TITLE
Problem with PAndF Kagi charts.

### ISSUE THREAD

**Issue by Libertaro** [rrag/react-stockcharts#305]
Hello. I have problems with these charts. 
1.**PAndF**. http://plnkr.co/edit/wje4qgiGLAmQPjd7SDWj?p=preview . Maybe something wrong with my data, but I receive it from Quandl. Chart works slower than usually and elements become too wide if you make zoom closer. 
2. **Kagi**. http://plnkr.co/edit/0XjP6g4byHIOooEr59cQ?p=preview. The same data and I receive error in console. 
Both charts I tried run with your test data, and it works perfectly. Other charts (**Area**, **Line**, **Candle**, **Renko**, **Heikin**) works with my data (which are in plunker) also correctly. 
There is an idea that in first case I should use parameter for example "reversal", but I can't use it properly.
Thanks for your feedback.

**Comment by rrag** [comment]
for p&F the problem is with the options, you have to set different boxsize, the default is 0.5
I changed it to
```
        var pAndF = pointAndFigure()
          .boxSize(20);
```

and it looks much better. Some time in the future I wish to automatically derive boxSize based on the price of the underlying security.

as for kagi. I am not sure what the problem is, let me check and get back to you

**Comment by rrag** [comment]
the kagi issue is a good one, this happens when the 0th index open and close are same

![deepinscreenshot_select-area_20170719220939](https://user-images.githubusercontent.com/10691183/28398873-090546b8-6ccf-11e7-898f-6a0bd64d7091.png)

as it is considered going up if `close > open`, and going down if `close < open` since it is neither it skips these conditions and forms a single line, I will fix it with going up is close >= open and that should solve it.

will publish a new 0.7 beta later this week with the fix.

**Comment by Libertaro** [comment]
@rrag thank you very much for answering so quickly. And Can you explain me please how to send parameters from index.html to chart.jsx for using them there? Should I use Babel for it ?

**Comment by rrag** [comment]
fixed in `0.7.0-beta.10`

**Comment by rrag** [comment]
@Libertaro not sure I follow, you dont have to send parameters from index.html to chart.jsx, you could check the recent examples on [codesandbox](https://github.com/rrag/react-stockcharts-examples2) for some inspiration
